tests/mock_network/requests.rs

```diff
@@ -56,13 +58,14 @@ fn successful_put_request() {
         }
     }
 
-    // TODO: Assert a quorum here.
+    assert!(request_received_count >= quorum);
     assert!(2 * request_received_count > min_section_size);
```

Comment:
I think we want to replace the old assertion with the new one as they check the same value

Reply:
Removed the original assertion, thanks!